src/gradNonLinearForm.cpp

```diff
@@ -34,7 +34,7 @@
 #include "riemann_solver.hpp"
 
 GradNonLinearForm::GradNonLinearForm(ParFiniteElementSpace *_vfes, IntegrationRules *_intRules, const int _dim,
-                                     const int _num_equation, const volumeFaceIntegrationArrays &_gpuArrays,
+                                     const int _num_equation, const precomputedIntegrationData &_gpuArrays,
```

Comment:
`precomputedIntegrationData` is passed in (replacing `volumeFaceIntegrationArrays`).  Check that it is actually used.  At first glance, I don't see it.  If not used, don't pass.

Reply:
fde5445